e2e-tests/storage/compare/statefulset_hostpath-pxc.yml

```diff
@@ -129,6 +131,8 @@ spec:
         imagePullPolicy: IfNotPresent
         name: pxc-init
         resources: {}
+        securityContext:
+          privileged: false
```

Comment:
hm, is it really two times exists in one yaml? looks like a duplicate in 108-109 lines

Reply:
really. Those lines were taken from kubernetes